eslint.config.js

```diff
@@ -0,0 +1,21 @@
+import js from "@eslint/js";
+import eslintConfigPrettier from "eslint-config-prettier";
+
+export default [
+  js.configs.recommended,
+  eslintConfigPrettier,
+  {
+    languageOptions: {
+      ecmaVersion: 2020,
+      sourceType: "module",
+      globals: {
+        node: "readonly",
+        jest: "readonly"
+      }
+    },
+    rules: {
+      "no-undef": "off"
+    },
+    ignores: ["/coverage/"]
```

Comment:
I found this out the hard way, this needs to go into its own config or the `ignores` only applies to this particular config object. So, it would be:

```
export default [
  { ignores: ["/coverage/"] },
  ...
]
```